python/lsst/ts/electrometer/controller.py

```diff
@@ -372,10 +376,14 @@ async def stop_scan(self):
         # assume 0.2 seconds per sample for now until the bug
         # affecting the integration time is fixed.
         # Rough tests showed 330 data   points takes ~4s
-        #
-        read_timeout = self.commander.timeout + self.scan_duration * 0.2
+        # Number of lines is approximately scan_duration over integration time
+        num_of_lines = self.scan_duration / self.integration_time
+        time_per_line = 0.0047
+        # Add extra time to read_timeout using num_of_lines times time per
+        # sample time (assumption with 330 samples take ~4 seconds) with
+        # approximately 30% overhead.
+        read_timeout = self.commander.timeout + ((num_of_lines * time_per_line) * 1.3)
```

Comment:
1.3 is a magic number. Please extract to a constant and add a doc string indicating what the value means. You probably want to move the comments just above this line there.